api/swagger/swagger.yaml

```diff
@@ -0,0 +1,429 @@
+swagger: '2.0'
+info:
+  description: Blog api
+  version: 1.0.0
+  title: Blog
+tags:
+  - name: posts
+    description: Posts related operations
+  - name: user
+    description: Operations about user
+  - name: auth
+    description: User authentication
+  - name: statistic
+    description: Statistic
+schemes:
+  - http
+paths:
+  /posts:
+    post:
+      tags:
+        - posts
+      summary: Add a new post to the store
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - in: body
+          name: body
+          description: Post object that needs to be added to the store
+          required: true
+          schema:
+            $ref: '#/definitions/Post'
+      responses:
+        201:
+          $ref: "#/responses/creted"  
+        400:
+          $ref: '#/responses/badRequest'
+        403:
+          $ref: '#/responses/forbidden'
+    get:
+      tags:
+        - posts
+      summary: Get all posts
+      produces:
+        - application/json
+      responses:
+        200:
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Post'
+          description: Posts
+        400:
+          $ref: '#/responses/badRequest'
+        403:
+          $ref: '#/responses/forbidden'
+        404:
+          $ref: '#/responses/notFound'
+  /posts/{id}:        
+    get:
+      tags:
+        - posts
```

Comment:
why do you need a tag? should it be posts or `post`?

Reply:
just for grouping purposes. it could be 'post' by the way